eng/common/scripts/Package-Properties.ps1

```diff
@@ -241,10 +280,18 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
         }
     }
 
+    # now pass along the set of packages we've identified, the diff itself, and the full set of package properties
+    # to locate any additional packages that should be included for validation
     if ($AdditionalValidationPackagesFromPackageSetFn -and (Test-Path "Function:$AdditionalValidationPackagesFromPackageSetFn")) {
```

Comment:
I assume this would be removed once we convert folks over.

Reply:
I _think_ this will have to stick around to cover a few of the more esoteric functions in `java`.

I will prove that out by actually updating the `language-settings` directly in the sync/eng branches created for this PR.

Reply:
I would like to understand what scenarios still need this after we do our conversations. If we don't have any or we can generalize them into common I'd like to remove this.